.rubocop.yml

```diff
@@ -5,6 +5,7 @@ inherit_from: .rubocop_todo.yml
 require:
   - rubocop-rails
   - rubocop-rspec
+  - rubocop-thread_safety
```

Comment:
Can now probably remove the explicit `-r rubocop-thread_safety` from `lint.rake`:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/rakelib/lint.rake#L10

Reply:
For some context - the only reason we couldn't put `rubocop-thread_safety` here before is because `codeclimate` consumes this `.rubocop.yml` config file and does not support the thread safety cop.

Reply:
Yep understood re codeclimate -- and good pickup on lint.rake